hardware/data_acquisition_units.py

```diff
@@ -19,7 +20,28 @@
 from ctypes import byref
 import queue
 import time
+from hardware import u
+import random
 
+class MockDAQ:
+
+    def __init__(self, instr_name = None):
+        if not instr_name:

```

Comment:
you can condense this to one line:
`self.name = "DAQ -NI9215 " if instr_name is None else instr_name`

Reply:
@shinjoe Is this considered best practice?

Reply:
Depends on the team. Pep8 seems silent on the issue. If you'd use a [ternary](https://docs.python.org/3/faq/programming.html#is-there-an-equivalent-of-c-s-ternary-operator]) in this case (as I would) then, yea I recommend the above. At the very least, I'd recommend an `is None` comparison versus the implicit truthyness above.